src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/UserServiceTest.kt

```diff
@@ -1,55 +1,109 @@
 package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service
 
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
 import org.mockito.Mockito.mock
 import org.mockito.kotlin.doReturn
 import org.mockito.kotlin.whenever
+import org.springframework.security.access.AccessDeniedException
 import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.client.manageusers.ManageUsersClient
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.client.manageusers.model.UserDetailsDto.AuthSource
 import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.helper.isEqualTo
 import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.helper.isNotEqualTo
 import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.helper.userDetails
-import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.helper.userEmail
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.helper.userEmailAddress
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.UserService.Companion.getClientAsUser
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.UserService.Companion.getServiceAsUser
 
 class UserServiceTest {
 
   private val manageUsersClient: ManageUsersClient = mock()
   private val userService = UserService(manageUsersClient)
 
   @Test
-  fun `getContactDetails should return ContactDetails for valid email username`() {
-    val username = "user@example.com"
+  fun `getServiceAsUser should return ContactDetails for valid email username`() {
```

Comment:
Just my opinion.

I generally avoid putting method names in the test description as will not be corrected if renamed/refactored.